app/repos/relationships/cdo.js

```diff
@@ -14,6 +14,14 @@ const cdoRelationship = sequelize => [
     model: sequelize.models.dog_breed,
     as: 'dog_breed'
   },
+  {
```

Comment:
I might be having a brain freeze here but just double-check that a cdo is returned when it doesn't have any breaches associated with it